packages/tasks/src/tasks/depth-estimation/spec/input.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
Note: depth estimation is not listed in https://huggingface.co/docs/api-inference/detailed_parameters + widget doesn't exist. Should we still list it in case we add it in the future?